WcaOnRails/Dockerfile

```diff
@@ -7,9 +6,9 @@ WORKDIR /app
 
 RUN apt-get update && apt-get install -y curl gnupg
 
-# Install Node 10
+# Install Node 14 LTS
 # From: https://github.com/nodesource/distributions
-RUN curl -sL https://deb.nodesource.com/setup_10.x | bash
+RUN curl -fsSL https://deb.nodesource.com/setup_lts.x | bash -
```

Comment:
For NodeJS, a lot of libraries depend on making sure that you're running a specific version of Node. As such, I'd prefer if you fixed this to be v14 instead of your current "download whatever happens to be LTS at the time of booting the Docker container" approach.

This is not a critical issue for the Ubuntu packages below (in particular, removing the mariadb-client version is fine) because there we don't depend on having a specific version be installed for our toolchain to work.